roles/network_plugin/calico/templates/calico-config.yml.j2

```diff
@@ -18,3 +18,47 @@ data:
 {% if inventory_hostname in groups['k8s-cluster'] and peer_with_router|default(false) %}
   as: "{{ local_as|default(global_as_num) }}"
 {% endif -%}
+  # Configure the MTU to use
```

Comment:
This CNI configuration whether as a sidecar or a initcontainer is not part of the manifests but is defined during install tasks which certainly makes sens with kubespray and should be kept.

Reply:
remove the code segment.